frontend/src/components/CoronaProvinceComponents/CoronaProvinceCard.js

```diff
@@ -30,10 +31,12 @@ export default function CoronaProvinceCard({province}) {
                     </li>
                 </DetailsList>
             </CoronaCard>}
+            <ActivityButton to={{pathname:"/activities", state: {coronaDetails: coronaDetails, province: province}}}>Was kann ich heute machen?</ActivityButton>
```

Comment:
Not a big fan of transporting data with location state, I prefer using just url parameters and do a fetch on the target page, this is much more robust when the user navigates directly to "/activities"

Reply:
me neither, but the user does not have an option to access this site directly as it is not included in navigation, and I need a few details so I thought I try something new 🤷🏻‍♀️